website/docs/docs/contributing/documenting-a-new-adapter.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Documenting a new adapter"
+id: "documenting-a-new-adapter"
+---
+
+If you've already [built an adapter](/website/docs/docs/contributing/building-a-new-adapter.md), and [tested your adapter]((/website/docs/docs/contributing/testing-a-new-adapter.md)), it's time to document it so the dbt community will know that it exists and how to use it!
+
+## General Guidelines
+
+### Single Source of Truth
+
+We have asked our partner adapter maintainers to use the [docs.getdbt.com repo](https://github.com/dbt-labs/docs.getdbt.com) (i.e. this site) as the single-source-of-truth for documentation rather than having to maintain the same set of information in three different places. Instead, we ask that the adapter repo's `README.md` and the dataplatform's documentation pages simply link to the corresponding page on this docs site.
```

Comment:
```suggestion
We asked our adapter maintainers to use the [docs.getdbt.com repo](https://github.com/dbt-labs/docs.getdbt.com) (i.e. this site) as the single-source-of-truth for documentation rather than having to maintain the same set of information in three different places. The adapter repo's `README.md` and the data platform's documentation pages should simply link to the corresponding page on this docs site.
```

Cleaning this up so it's more direct. Also should we be linking to the repo? I think it makes sense but also sounds a bit odd on the docs site that is created from the repo.